tests/unit/lifecycle/LifecycleConductor.spec.js

```diff
@@ -242,19 +242,19 @@ describe('Lifecycle Conductor', () => {
     });
 
     describe('_indexesGetOrCreate', () => {
-        it('should return v1 for bucketd bucket sources', () => {
+        it('should return v2 for bucketd bucket sources', () => {
```

Comment:
should not have changed : `bucketd` sources (i.e. s3c) should still use lifecycle v1 tasks....

Reply:
The initial test case was false, there was a bug in the code that was fixed during the unification (with the previous code, S3C never used the v2 listing) the tests were missed as they didn't fail after the change  as they never called done() after `_indexesGetOrCreate` finished executing.

Bumping the "assert" package made the tests fail, not sure what the exact reason is, might be a performance increase or something else but now the assert function returns before the test ends.

Reply:
OK, but can we double-check with hopocalipse that we have an issue in the tests (i.e. bucketd must indeed use V2), or is there an issue in the lib (i.e. we should use V1 for bucketd, but we use V2) : maybe s3c should not use v2 listing, and it was done on purpose... aka activating v2 listing in s3c should not be a silent improvement thanks to the unification, but a conscious choice of which everybody should be aware.

Reply:
In the 7.x branch, listing v1 is only used when forcing it using the `forceLegacyListing` flag in the config. We kept the same behaviour in 9.x